hamocc/accfields.F90

```diff
@@ -90,22 +90,22 @@ SUBROUTINE ACCFIELDS(kpie,kpje,kpke,pdlxp,pdlyp,pddpo,omask)
       use mo_sedmnt,      only: powtra,sedlay,burial
       use mo_vgrid,       only: dp_min
 
-      implicit none
+      IMPLICIT NONE
```

Comment:
Fine for me to have MODULE, SUBROUTINE, etc. still in capital letters, since they indicate program units, but I don't understand the logic behind having IMPLICIT NONE in capital letters?